integration/SimpleCacheClient.test.ts

```diff
@@ -67,103 +74,100 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
     await momento.deleteCache(cacheName);
   });
-  // TODO: deprecating credentials file for now
-  // it('should use the default auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials();
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //
-  //   removeSystemCredentials();
-  // });
-  //
-  // it('should use the MOMENTO_PROFILE auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials('profile2');
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //   removeSystemCredentials();
-  // });
+
+  it('should use the default auth token from ~/.momento-test/credentials', async () => {
+    createSystemCredentials();
+    const cacheName = v4();
+    const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
+    await momento.createCache(cacheName);
+    await expect(momento.createCache(cacheName)).rejects.toThrow(
+      AlreadyExistsError
+    );
+    await momento.deleteCache(cacheName);
+
+    removeSystemCredentials();
+  });
+
+  it('should use the MOMENTO_PROFILE auth token from ~/.momento-test/credentials', async () => {
+    createSystemCredentials('profile2');
+    const cacheName = v4();
+    const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
+    await momento.createCache(cacheName);
+    await expect(momento.createCache(cacheName)).rejects.toThrow(
+      AlreadyExistsError
+    );
+    await momento.deleteCache(cacheName);
+    removeSystemCredentials();
+  });
 
   it('should create 1 cache and list the created cache', async () => {
-    const cacheName1 = v4();
+    const cacheName = v4();
     const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
-    await momento.createCache(cacheName1);
+    await momento.createCache(cacheName);
     const caches = (await momento.listCaches()).getCaches();
     const names = caches.map(c => c.getName());
-    expect(names.includes(cacheName1)).toBeTruthy();
-    await momento.deleteCache(cacheName1);
+    expect(names.includes(cacheName)).toBeTruthy();
+    await momento.deleteCache(cacheName);
   });
 
   it('should set and get string from cache', async () => {
-    const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
     const cacheKey = v4();
     const cacheValue = v4();
-    await momento.set(INTEGRATION_TEST_CACHE_NAME, cacheKey, cacheValue);
-    const res = await momento.get(INTEGRATION_TEST_CACHE_NAME, cacheKey);
+    await momento2.set(INTEGRATION_TEST_CACHE_NAME, cacheKey, cacheValue);
+    const res = await momento2.get(INTEGRATION_TEST_CACHE_NAME, cacheKey);
```

Comment:
Why is this `momento2`? If it can't be `momento` could it be something a bit more obvious? Not sure I have a great idea for a name but like `default_momento` or something even? But I think it should just be able to be `momento`??

(Same comments also for the rest of the file)

Reply:
Ah I think I was trying out different testing patters and forgot to update that naming.
Thank you for catching it!